### DISCUSSION TITLE
Show account creation in chat

### DISCUSSION THREAD

**Discussion by MiClobiMC** [night/betterttv#5863] [Ideas]
It would be very useful if one could configure so that the account creation is shown when users speak in the chat. Because it is a bit tedious to have to see the creation of the account when you open the profile, it would be better to see it when they write

**Comment by night** [comment]
Thanks for the suggestion. Since Twitch does not expose this information in chat messages we would need to do many individual calls to their API for this information, which is simply not practical at scale. Because of this, it is not a feature we can consider at this time.
